bindings/nodejs/README.md

```diff
@@ -1,46 +1,109 @@
 # IOTA SDK Library - Node.js binding
 
+## Table of contents
+
+- [Requirements (only for building the binary yourself)](#requirements-only-for-building-the-binary-yourself)
+- [Getting Started](#getting-started)
+    - [Install the IOTA SDK](#install-the-iota-sdk)
+    - [Usage](#usage)
+      - [Wallet](#wallet)
+      - [Client](#client)
+      - [Examples](#examples)
+- [Available Scripts](#available-scripts)
+    - [`npm install`](#npm-install)
+    - [`npm build`](#npm-build)
+    - [`npm test`](#npm-test)
+    - [Cargo.toml](#cargotoml)
+    - [README.md](#readmemd)
+    - [index.node](#indexnode)
+    - [package.json](#packagejson)
+    - [src/](#src)
+    - [src/lib.rs](#srclibrs)
+    - [target/](#target)
+- [Learn More](#learn-more)
+
 ## Requirements (only for building the binary yourself)
 
-Ensure you have first installed the required dependencies for the library [here](https://github.com/iotaledger/iota-sdk/tree/develop/sdk#readme) and on Windows also LLVM, our workflow uses `https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe`. On Windows, you may also need to set an environment variable `RUSTFLAGS` to `-C target-feature=+crt-static`.
+Please ensure you have installed the [required dependencies for the library for Rust code](/README.md#requirements), as
+well as the following:
 
 - Python < 3.11
```

Comment:
is this tested? Otherwise i think we should just say python 3

Reply:
Below we say: `[Python 3.x](https://www.python.org)`

Reply:
Doesn't work with Python 3.11. I changed that when I started working on the examples as my default python version was 3.11